osmaxx/api_client/API_client.py

```diff
@@ -90,5 +91,17 @@ def _authorization_headers(self):
         }
 
 
+class LazyChunkedRemoteFile:
+    def __init__(self, *args, download_fu=requests.get, **kwargs):
```

Comment:
Maybe document why this works, otherwise we ask ourselves this question next time ;-)